client/state/stats/lists/utils.js

```diff
@@ -172,6 +172,33 @@ export function parseOrderDeltas( payload ) {
 	} );
 }
 
+/**
+ * Create the correct property and value for a label to be used in a chart
+ *
+ * @param {string} unit - day, week, month, year
+ * @param {object} date - moment object
+ * @param {object} localizedDate - moment object
+ * @return {object}
+ */
+function getChartLabels( unit = '', date, localizedDate ) {
```

Comment:
should the unit default be set to a valid option here like `day`?

Reply:
Changing this to return and empty object, similar to if an invalid date is passed. I think its better to show no label than an incorrect one.